src/scripts/send-magic-link.ts

```
@@ -0,0 +1,31 @@
+import { createClient } from '@supabase/supabase-js'
+
+// Replace with your Supabase URL and anon key
+const supabaseUrl = process.env.VITE_SUPABASE_URL!
```

ESLint

[error] Forbidden non-null assertion.

(@typescript-eslint/no-non-null-assertion)